### DISCUSSION TITLE
Setting correct path for Projects referenced using ProjectReferences where the Project is not part of the template

### DISCUSSION THREAD

**Discussion by Cjewett** [dotnet/templating#7508] [Template authoring]
We want to create templates that have references to an existing project like so: `<ProjectReference` Include="..\..\..\SomeProject.csproj" />`. These existing projects are not part of the template. After installing the template and creating a new project relative paths don't point at SomeProject.csproj correctly due to the relative pathing being different based on where you created the project. I was wondering if there was a recommended way to solving this problem?

Thanks

**Comment by david-acm** [comment]
@Cjewett I was facing the same issue which I documented [#7833](https://github.com/dotnet/templating/discussions/7833). I opened this PR [#40133](https://github.com/dotnet/sdk/pull/40133) with an enhancement to support this use case. It is already approved 😃, so vote it up, hopefully the .Net team will merge it soon.

For now, the only workaround i know is to run a `dotnet add [ExistingProject] reference [CreatedProject]` command. You could add an error message suggesting the user to run this themselves.

**Reply by david-acm** [reply]
Btw this was approved, merged and available since .Net SDK 9.0.100.prev6
